app/services/food_service.rb

```diff
@@ -0,0 +1,19 @@
+class FoodService
+  def self.foods_by_ingredient(ingredient)
+    ingredient_search = ingredient.gsub(/\s+/, "")
```

Comment:
This shouldn't be necessary. Faraday will do this for us. You can pass `ingredient` directly to the `get_data` method on line 4 and it should work!